### PR TITLE
added MetricsName to keep common metric names

### PR DESCRIPTION
**Pull request by shabd67**
# Resolves

[SALUS-1008](https://jira.rax.io/browse/SALUS-1008)

# What

added MetricsName to keep common metric names

### CODE REVIEW

File: src/main/java/com/rackspace/salus/common/config/MetricsName.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.rackspace.salus.common.config;
+
+public interface MetricsName {
```
Comment (itzg) [inline src/main/java/com/rackspace/salus/common/config/MetricsName.java:19]: nit:
```suggestion
public interface MetricsNames {
```

**Review by itzg — CHANGES_REQUESTED** [review]
With the pre-review of monitor management, it looks like you could also add commonly used tag keys (and even tag values), such as "operation" and "service". And then for "operation" there would be some common tag values like "create", "update", "delete".

**Review by itzg — DISMISSED** [review]

**Review by jjbuchan — COMMENTED** [review]
Was there other discussion I missed around @itzg 's previous suggestion about adding constants for the operations?

**Comment by itzg** [comment]
> Was there other discussion I missed around @itzg 's previous suggestion about adding constants for the operations?

Yeah, our comments kind of conflicted :) but I'm good with introducing an adjacent class to hold the names of tag keys and values. Like `MetricTags`?

File: src/main/java/com/rackspace/salus/common/config/MetricTagsValues.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.rackspace.salus.common.config;
+
+public interface MetricTagsValues {
```
Comment (itzg) [inline src/main/java/com/rackspace/salus/common/config/MetricTagsValues.java:19]:
```suggestion
public interface MetricTagValues {
```

**Review by itzg — CHANGES_REQUESTED** [review]
Just a nit

**Review by itzg — APPROVED** [review]

**Pull request merged**